<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>771</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Task 1</string>
     </attribute>
     <widget class="QPlainTextEdit" name="tab1_plainTextEdit">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="tab1_clearButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>350</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tab1_correctButton">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>350</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Correct</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Task 2</string>
     </attribute>
     <widget class="QListView" name="tab2_listView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>431</width>
        <height>331</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="tab2_lineEdit">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>350</y>
        <width>181</width>
        <height>28</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="tab2_pushButton_Del">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>350</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tab2_pushButton_UP">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>10</y>
        <width>51</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>UP</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tab2_pushButton_DOWN">
      <property name="geometry">
       <rect>
        <x>450</x>
        <y>50</y>
        <width>51</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>DOWN</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tab2_pushButton_NewEdit">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>350</y>
        <width>71</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>New</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tab2_pushButton_Save">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>350</y>
        <width>61</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="tab2_checkBox_IconView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>390</y>
        <width>93</width>
        <height>26</height>
       </rect>
      </property>
      <property name="text">
       <string>Icon View</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Task 3</string>
     </attribute>
     <widget class="QTableView" name="tab3_tableView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>541</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="tab3_pushButton_Select">
      <property name="geometry">
       <rect>
        <x>330</x>
        <y>420</y>
        <width>83</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Select</string>
      </property>
     </widget>
     <widget class="QPushButton" name="tab3_pushButton_ClearSel">
      <property name="geometry">
       <rect>
        <x>432</x>
        <y>420</y>
        <width>121</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear Selection</string>
      </property>
     </widget>
     <widget class="QComboBox" name="tab3_comboBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>410</y>
        <width>231</width>
        <height>28</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>red</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>green</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>blue</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>cyan</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>magenta</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>yellow</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>gray</string>
       </property>
      </item>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
